tests/saml/controller_test.py

```diff
@@ -0,0 +1,18 @@
+from api.saml.provider import SAMLWebSSOAuthenticationProvider
+from core.model import ExternalIntegration
+from tests.test_controller import ControllerTest as BaseControllerTest
+
+
+class ControllerTest(BaseControllerTest):
+    def __init__(self):
+        self._library = None
+        self._integration = None
+
+    def setup(self, _db=None, set_up_circulation_manager=True):
+        super(ControllerTest, self).setup(_db, set_up_circulation_manager)
+
+        self._library = self.make_default_library(self._db)
```

Comment:
It shouldn't be necessary to call make_default_library; it will be created the first time you refer to self._default_library.

Reply:
Fixed it, thank you